aws/eks/tests/test_setup_lenses.py

```diff
@@ -0,0 +1,46 @@
+import yaml
+import json
+import sys
+import os
+
+from setup_lenses import SetupLenes
+configure_lenses = SetupLenes()
+
+class TestSetupLenses():
+
+    # def test_create_license(self):
+    #     lenses_license = {
+    #         "source":"Lenses.io Ltd",
+    #         "clientId":"ClinetTest",
+    #         "details":"Lenses",
+    #         "key":"lenses_key"
+    #     }
+
+    #     # This always fails because we do not json.dump the dict
+    #     self.create_license, err = configure_lenses.CreateLensesLicense(
+    #         secret=lenses_license
+    #     )
+    #     assert err != 0
+
+    def test_create_manifest(self):
+        brokers = "PLAINTEXT://10.164.0.9:51092"
+        zookeepers =  "{url:\"10.164.0.9:51181\",jmx:\"10.164.0.9:51585\"}"
+        lenses_admin_username = "admin"
+        lenses_admin_password = "admin"
+        kafka_metrics_opts = ["{id: 1,  url:\"http://10.164.0.9:11001/metrics\"}"]
+
+        lenses_manifest, err = configure_lenses.CreateLensesManifest(
+            brokers=brokers,
+            zookeepers=zookeepers,
+            username=lenses_admin_username,
+            password=lenses_admin_password,
+            kafka_metrics_opts=kafka_metrics_opts
+        )
+        assert err == 0
```

Comment:
should you check there also if the values are included correctly?

Reply:
@spirosoik unit tests for the values will be included when new code of 309 is pushed